app/views/get_ready_for_brexit_checker_requests/_request_details.html.erb

```diff
@@ -29,4 +29,57 @@
   ) %>
 </div>
 
+<div id="add_action">
+  <h2>Add a new action</h2>
+  <%= f.input(
+    :add_audience,
+    label: "Who needs to know?",
+    hint: "Specify who, where they are, and what they’re doing. For example, an Irish citizen who lives in the UK taking a pet to the EU.",
+  ) %>
+
+  <%= f.input(
+    :add_action,
+    label: "What is the action?",
+    hint: "What does the user need to do to prepare for no deal Brexit?",
+    ) %>
+
+  <%= f.input(
+    :add_consequence,
+    label: "Consequence",
+    hint: "What is the problem for the user if they don’t act?",
+    ) %>
+
+  <%= f.input(
+    :add_service_link,
+    label: "Service link",
+    hint: "Provide a link to any application process or service, if there is one.",
+    ) %>
+
+  <%= f.input(
+    :add_guidance_link,
+    label: "Guidance link",
+    hint: "Provide a link to the page that explains things in detail. It does not have to be on GOV.UK.",
+    ) %>
+
+  <%= f.input(
+    :add_lead_time,
+    label: "Lead time",
+    hint: "How long does the process take, if applicable?",
+    ) %>
+
+  <%= f.input(
+    :add_deadline,
+    label: "Deadline",
+    hint: "Does the user need to act before a certain date?",
+    ) %>
```

Comment:
Do we need an input for priority?

Reply:
No, the default in Zendesk is 'normal' and we don't have a requirement to change it to another level.

Reply:
Ah sorry, I mean for the new action.

Reply:
Ah I see - will check with Ben.

Reply:
Priority field has been added 👍